scripts/docker/rocky8-epel-rpm.Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM rockylinux:8
+
+ARG artifact_url=""
+ADD ${artifact_url} /tmp
+ADD node_modules /usr/share/mongodb-csfle-library-version/node_modules
+RUN dnf update -y
+# epel-release so that openssl3 is installable
```

Comment:
I considered just adding this to the regular Rocky Linux 8 Dockerfile, but decided against it since it’s nice to actually know that no special packages are required for install the static-openssl mongosh on the base system.